From bb7e86acfb308cc4fce88712e755a461a97c57dd Mon Sep 17 00:00:00 2001
From: Baptiste Assmann <bedis9@gmail.com>
Date: Wed, 3 Sep 2014 18:29:47 +0200
Subject: [PATCH 82/82] DOC: missing track-sc* in http-request rules

track-sc is well defined in http-request rules, but not listed in
option list.
This patch fix this miss.
---
 doc/configuration.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/doc/configuration.txt b/doc/configuration.txt
index f7776a9..c0aeb3a 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -2916,7 +2916,8 @@ http-request { allow | deny | tarpit | auth [realm <realm>] | redirect <rule> |
               add-acl(<file name>) <key fmt> |
               del-acl(<file name>) <key fmt> |
               del-map(<file name>) <key fmt> |
-              set-map(<file name>) <key fmt> <value fmt>
+              set-map(<file name>) <key fmt> <value fmt> |
+              { track-sc0 | track-sc1 | track-sc2 } <key> [table <table>]
              }
              [ { if | unless } <condition> ]
   Access control for Layer 7 requests
-- 
1.8.5.5

